controllers/tasks.js

```diff
@@ -347,6 +347,32 @@ const updateTaskStatus = async (req, res, next) => {
           return res.boom.badRequest("Status cannot be updated. Task is not done yet");
```

Comment:
can we have better message

Reply:
Updated the msg in PR https://github.com/Real-Dev-Squad/website-backend/pull/1937